install_goto.sh

```diff
@@ -8,7 +8,29 @@ touch ${HOME}/.goto/active-project
 ln -s /usr/local/opt/goto/bin/* /usr/local/bin/
 
 # add init_script to.bash_profile:
-echo "source start_goto" >> ${HOME}/.bash_profile
+
+echo "Goto needs to run a startup script for it to work. "
+echo "This can be automatically added now to your .bash_profile file"
+echo "If you want to do this manually add the line 'source start_goto'"
+
+if [ -f "${HOME}/.bash_profile" ]; then
+    read -p "Want to automatically add the goto startup script to your .bash_profile? " -n 1 -r
```

Comment:
Want to automatically ~add~  **append**  `source start_goto` to your `.bash_profile`? (required for goto to work correctly)

Reply:
or maybe just keep it simple ... yeah 👍

Reply:
Ignore this suggestion. See discussion below.